upgrade_test.py

```diff
@@ -769,6 +766,8 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
                 self.fill_and_verify_db_data('after upgraded %s' % self.db_cluster.node_to_upgrade.name)
                 self.search_for_idx_token_error_after_upgrade(node=self.db_cluster.node_to_upgrade,
                                                               step=step)
+        InfoEvent(message='Step5.1 - run raft topology upgrade procedure').publish()
+        self.run_raft_topology_upgrade_procedure()
```

Comment:
reft upgrade is always triggered - what will happen in 1 month when we release 6.1 and we'll upgrade from 6.0 with raft already enabled? Shouldn't we protect from upgrading when not needed?

Reply:
it will do the same, just check that feature is supported by all nodes, and check the api response. Attempt to trigger raft upgrade procedure second time will not affect the cluster, it just finished faster